prepare_kb.py

```diff
@@ -0,0 +1,10 @@
+import os
+from scripts.copy_kb import copy_kb
+
+kb_path = os.path.dirname(os.path.abspath(__file__))
+ostis_path = os.path.split(kb_path)[0]
+copy_path = os.path.join(ostis_path, 'ims.ostis.kb_copy')
+copy_kb(kb_path, copy_path)
+for script in os.listdir(os.path.join(kb_path, 'scripts')):
+    if os.path.isfile(os.path.join(kb_path, 'scripts', script)) and script != 'copy_kb.py' and script.endswith('.py'):
+        os.system("python3 " + os.path.join(kb_path, 'scripts', script) + ' ' + copy_path)
```

Comment:
I think we will need to have ability to run scripts in a given order or not run some script. Simple way do support it is create list of scripts here and run each script from this list in turn